### PR TITLE
net-p2p/syncthing-relaysrv: new ebuild

### PR DESCRIPTION
**Pull request by khumarahn**
This is an ebuild for syncthing relay server https://github.com/syncthing/relaysrv. Such relays connect computers which are unable to establish direct connection because of NATs or firewalls.

The ebuild is simple and seems to work for me. Would be nice if somebody could test it as well.

### CODE REVIEW

File: net-p2p/syncthing-relaysrv/syncthing-relaysrv-0.12.18.ebuild [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+USER=syncthing-relaysrv
+BINARY=syncthing-relaysrv
+
+pkg_setup() {
+	enewgroup ${USER}
+	enewuser ${USER} -1 -1 /var/lib/${USER} ${USER}
+}
+
+src_compile() {
+	cd "src/${EGO_PN}"
```
Comment (monsieurp) [inline net-p2p/syncthing-relaysrv/syncthing-relaysrv-0.12.18.ebuild]: missing || die

File: net-p2p/syncthing-relaysrv/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
```
Comment (monsieurp) [inline net-p2p/syncthing-relaysrv/metadata.xml]: Add yourself to this file as well please.

**Comment by monsieurp** [comment]
@gentoo/proxy-maint

File: net-p2p/syncthing-relaysrv/syncthing-relaysrv-0.12.18.ebuild (hunk 1 above)
Comment (khumarahn) [inline net-p2p/syncthing-relaysrv/syncthing-relaysrv-0.12.18.ebuild]: corrected

File: net-p2p/syncthing-relaysrv/metadata.xml (hunk 2 above)
Comment (khumarahn) [inline net-p2p/syncthing-relaysrv/metadata.xml]: added

**Pull request closed without merging**